### ISSUE TITLE
Remove Usages of Reserved Word 'class'

### ISSUE THREAD

**Issue by feltnerm** [krispo/angular-nvd3#110]
According to the ECMAScript specification `class` is a reserved word. In ES5.1 it was ["future reserved word"][es5.1] so I think most browsers let it go, but unfortunately IE9 actually got this one right and barfs when `class` is used. In the most recent version of ECMAScript (6) `class` is [definitely a reserved word][es6].

There are a [few][code-example-1] [places][code-example-2] in the code where this word is used.

I literally just dove into `angular-nvd3` today to evaluate its potential for a project, and discovered this when doing some IE9 testing. I don't feel as if I understand the entire scope of a change like this, and was hoping someone would be willing to chime in. It seems, at first glance, that a change like this could result in breaking changes for end-users.

That being said, I'm willing to put together a pull-request if someone is willing to review. Simply changing the variable to be `className` (or something else) should fix this issue.

tl;dr - `class` is a reserved word in ECMAScript. It makes IE9 barf. Can we fix it? 

[es5.1]: http://www.ecma-international.org/ecma-262/5.1/#sec-7.6.1.2
[es6]: https://people.mozilla.org/~jorendorff/es6-draft.html#sec-reserved-words
[code-example-1]: https://github.com/krispo/angular-nvd3/blob/5355987fb95ffb53d6753093bfa093fb433c50ba/src/angular-nvd3.js#L237
[code-example-2]: https://github.com/krispo/angular-nvd3/blob/5355987fb95ffb53d6753093bfa093fb433c50ba/src/angular-nvd3.js#L269

**Comment by NameFILIP** [comment]
Thank you, @feltnerm!
